src/main/java/com/code_intelligence/jazzer/driver/FuzzTargetRunner.java

```diff
@@ -64,9 +65,6 @@ public final class FuzzTargetRunner {
 
   private static final long BYTE_ARRAY_OFFSET = UNSAFE.arrayBaseOffset(byte[].class);
 
-  // Default value of the libFuzzer -error_exitcode flag.
-  private static final int LIBFUZZER_ERROR_EXIT_CODE = 77;
-
   // Possible return values for the libFuzzer callback runOne.
   private static final int LIBFUZZER_CONTINUE = 0;
   private static final int LIBFUZZER_RETURN_FROM_DRIVER = -2;
```

Comment:
I only extracted this one because it's relevant outside `FuzzTargetRunner` and needed elsewhere. If the other constants aren't needed elsewhere, I would prefer to keep them private to this class.